diff -rN -u old-clbuild.mystic/my-dependencies new-clbuild.mystic/my-dependencies
--- old-clbuild.mystic/my-dependencies	2009-01-17 17:18:08.304268462 +0100
+++ new-clbuild.mystic/my-dependencies	2009-01-17 17:18:08.310935806 +0100
@@ -2,7 +2,9 @@
 qbook arnesi iterate cl-ppcre yaclml
 yaclml arnesi iterate 
 local-time cl-fad
-contextl portable-threads lw-compat
+contextl portable-threads lw-compat trivial-garbage
+cxml closure-common
+cl-l10n cxml
 
 # systems that reside in mystic/packages and are missing from clbuild: trivial-email-utf-8 cl-smtp cl-i18n 
 # systems that reside somewhere else: swank 
@@ -11,7 +13,7 @@
 # some other unknown dependencies: postmodern(put under clbuild)
 # dependencies which could be disabled due to private modifications: elephant weblocks. We should set them initially to collect all deps, but then
 # we can disable them 
-mystic alexandria anaphora arnesi asdf-system-connections babel bordeaux-threads cffi chunga cl+ssl cl-base64 cl-cont cl-containers cl-fad cl-json cl-l10n cl-ppcre cl-unicode cl-who closer-mop clsql contextl elephant f-underscore fare-matcher fare-utils fiveam flexi-streams fset html-template hunchentoot ieee-floats ironclad iterate lift local-time log5 lw-compat md5 metabang-bind metatilities metatilities-base misc-extensions moptilities parenscript portable-threads postmodern puri rfc2388 rt split-sequence trivial-features trivial-gray-streams trivial-utf-8 url-rewrite usocket weblocks
+zzzmystic alexandria anaphora arnesi asdf-system-connections babel bordeaux-threads cffi chunga cl+ssl cl-base64 cl-cont cl-containers cl-fad cl-json cl-l10n cl-ppcre cl-unicode cl-who closer-mop clsql contextl elephant f-underscore fare-matcher fare-utils fiveam flexi-streams fset html-template hunchentoot ieee-floats ironclad iterate lift local-time log5 lw-compat md5 metabang-bind metatilities metatilities-base misc-extensions moptilities parenscript portable-threads postmodern puri rfc2388 rt slime split-sequence trivial-features trivial-gray-streams trivial-utf-8 url-rewrite usocket weblocks
 
 
 
diff -rN -u old-clbuild.mystic/wnpp-projects new-clbuild.mystic/wnpp-projects
--- old-clbuild.mystic/wnpp-projects	2009-01-17 17:18:08.304268462 +0100
+++ new-clbuild.mystic/wnpp-projects	2009-01-17 17:18:08.310935806 +0100
@@ -152,7 +152,7 @@
 yaclml       get_darcs http://common-lisp.net/project/bese/repos/yaclml #generating XML/HTML
 log5 get_darcs http://common-lisp.net/project/log5 # logging framework organized around five things: categories, outputs, senders, messages and contexts
 # cl-randist get_git http://lambdatau.com/git/cl-randist.git # random number distribution. Repo fails, download from http://code.google.com/p/cl-randist/
-contextl get_darcs http://common-lisp.net/project/closer/repos/contextl/  # CLOS extension for Context-oriented Programming
+contextl get_darcs http://common-lisp.net/project/closer/repos/contextl  # CLOS extension for Context-oriented Programming
 cl-l10n get_darcs http://www.common-lisp.net/project/cl-l10n/repos/cl-l10n # localization
 local-time get_darcs http://common-lisp.net/project/local-time/darcs/local-time # library for manipulating date and time information
 trivial-email-utf-8 get_darcs http://common-lisp.net/project/bpm/darcs/trivial-email-utf-8 # http://groups.google.com/group/cl-terrace/web/trivial-email-utf-8

